### PR TITLE
Feature GET scenario

### PR DESCRIPTION
**Pull request by cristianfchq**

### CODE REVIEW

File: src/test/java/account/AccountTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Test(groups = {"createAccount", "deleteAccount"})
+    public void getAccountTest() {
+        requestBuilder
+                .addEndpoint("/Account/{accountID}")
+                .addPathParams("accountID", responseObject.getId())
+                .addMethod(ApiMethod.GET)
+                .build();
+        apiResponse = ApiManager.execute(requestBuilder.build());
+//        Account account = apiResponse.getBody(Account.class);
```
Comment (JessickaMoya) [inline src/test/java/account/AccountTest.java]: There shouldn't be commented lines

File: src/test/java/account/AccountTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+//        System.out.println("------------ " + account.getName());
+        Assert.assertEquals(apiResponse.getStatusCode(), 200);
+        System.out.println("++++++++++++" + apiResponse.getStatusCode());
```
Comment (JessickaMoya) [inline src/test/java/account/AccountTest.java:20]: There shouldn't be print lines

File: src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java [hunk 3]
```diff
@@ -35,14 +74,25 @@ public void deleteContract() {
...
     public void iExecuteTheGetSingleContractRequestOnEndpoint(final String endpoint) {
+        requestBuilder
+                .addEndpoint("/Contract/{contractID}")
+                .addPathParams("contractID", responseObject.getId())
+                .build();
+        apiResponse = ApiManager.execute(requestBuilder.build());
     }
 
     @Then("The response status code should be {string} on get single Contract request")
     public void theResponseStatusCodeShouldBeOnGetSingleContractRequest(final String statusCode) {
```
Comment (Raymundogs) [inline src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java:134]: parameter statusCode is not used. If it is not necesary, it should be deleted.

File: src/test/java/account/AccountTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        accountToSend.setName("cristian choque prueba desde java actualizado");
+        requestBuilder
+                .addEndpoint("/Account/{accountID}")
+                .addPathParams("accountID", responseObject.getId().toString())
+                .addBody(new ObjectMapper().writeValueAsString(accountToSend))
+                .addMethod(ApiMethod.PATCH)
+                .build();
+        apiResponse = ApiManager.executeWithBody(requestBuilder.build());
+        Assert.assertEquals(apiResponse.getStatusCode(), 204);
+        System.out.println("++++++++++++" + apiResponse.getStatusCode());
```
Comment (JessickaMoya) [inline src/test/java/account/AccountTest.java:34]: There shouldn't be print lines

File: src/test/java/account/AccountTest.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+    @Test(groups = "createAccount")
+    public void deleteAAccount() {
+        requestBuilder
+                .addEndpoint("/Account/{accountID}")
+                .addPathParams("accountID", responseObject.getId().toString())
+                .addMethod(ApiMethod.DELETE)
+                .build();
+        apiResponse = ApiManager.execute(requestBuilder.build());
+//        Account account = apiResponse.getBody(Account.class);
```
Comment (JessickaMoya) [inline src/test/java/account/AccountTest.java]: There shouldn't be commented lines

File: src/test/java/account/AccountTest.java [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        requestBuilder
+                .addEndpoint("/Account/")
+                .addBody(new ObjectMapper().writeValueAsString(accountToSend))
+                .addMethod(ApiMethod.POST)
+                .build();
+        ApiResponse apiResponse = ApiManager.executeWithBody(requestBuilder.build());
+        responseObject = apiResponse.getBody(ResponseObject.class);
+        System.out.println("----------------" + responseObject.getId());
+        Assert.assertEquals(apiResponse.getStatusCode(), 201);
+//        Assert.assertEquals(account.getAttributes().getType(), "Account");
```
Comment (JessickaMoya) [inline src/test/java/account/AccountTest.java]: There shouldn't be commented lines

File: src/test/java/account/AccountTest.java [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    public void createAAccountTest() throws JsonProcessingException {
+        accountToSend.setName("cristian choque prueba desde java 2");
+        requestBuilder
+                .addEndpoint("/Account/")
+                .addBody(new ObjectMapper().writeValueAsString(accountToSend))
+                .addMethod(ApiMethod.POST)
+                .build();
+        ApiResponse apiResponse = ApiManager.executeWithBody(requestBuilder.build());
+        responseObject = apiResponse.getBody(ResponseObject.class);
+        System.out.println("----------------" + responseObject.getId());
```
Comment (JessickaMoya) [inline src/test/java/account/AccountTest.java]: There shouldn't be print lines

File: src/test/java/rest/salesforce/org/steps/OrdersApiSteps.java [hunk 8]
```diff
@@ -34,14 +73,25 @@ public void deleteOrder() {
...
     @Given("I build a {string} request for a single Order")
     public void iBuildARequestForASingleOrder(final String apiMethod) {
+        requestBuilder
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addMethod(ApiMethod.GET);
     }
 
     @When("I execute the get single Order request on {string} endpoint")
     public void iExecuteTheGetSingleOrderRequestOnEndpoint(final String endpoint) {
```
Comment (Raymundogs) [inline src/test/java/rest/salesforce/org/steps/OrdersApiSteps.java:149]: Parameter endpoint is never used, it should be deleted.

File: src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java [hunk 9]
```diff
@@ -9,20 +9,56 @@
...
+
+    private ApiRequestBuilder requestBuilder = new ApiRequestBuilder();
+    private ResponseObject responseObject = new ResponseObject();
+    private ApiResponse apiResponse = new ApiResponse();
+    Account accountToSend = new Account();
+
     @Before("@CreateAndDeleteAccount")
-    public void createAnAccount() {
+    public void createAnAccount() throws JsonProcessingException {
+        accountToSend.setName("cristian choque desde java");
```
Comment (JessickaMoya) [inline src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java]: change the name given to a more general name, and use english

Comment (cristianfchq) [reply src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java]: ok

File: src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java (hunk 3 above)
Comment (cristianfchq) [reply src/test/java/rest/salesforce/org/steps/ContractsApiSteps.java:134]: ok

File: src/test/java/rest/salesforce/org/steps/OrdersApiSteps.java (hunk 8 above)
Comment (cristianfchq) [reply src/test/java/rest/salesforce/org/steps/OrdersApiSteps.java:149]: ok

**Review by Raymundogs — APPROVED** [review]

**Review by JessickaMoya — APPROVED** [review]

**Pull request merged**